### DISCUSSION TITLE
Enable score submission for unranked maps?

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#15502] [Q&A] [answered]
Since stable does it, I thought it'd make sense for lazer to do it too.

**Comment by peppy** [comment, marked as answer]
Please include logs and a beatmap link showing that this is not happening

**Reply by Flamiii** [reply]
I'm not OP, but I can say that after having played osu!lazer for the past few weeks, I've been experiencing the same issue of plays on unranked maps not submitting. I just assumed that this was intentional behavior, but maybe not? Either way, I just played this [unranked map](https://osu.ppy.sh/beatmapsets/217105#osu/509346) a few minutes ago and there seems to be no submitted score on my profile from playing it.

[database.log](https://github.com/ppy/osu/files/7491597/database.log)
[network.log](https://github.com/ppy/osu/files/7491598/network.log)
[performance.log](https://github.com/ppy/osu/files/7491599/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7491600/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7491601/updater.log)

**Reply by peppy** [reply]
Thanks. Tracking this issue as an osu-web oversight https://github.com/ppy/osu-web/issues/8301.

**Reply by integerrr** [reply]
?! Was wondering why discord bots for recent scores stopped working for unranked, thought  that was intentional

**Reply by peppy** [reply]
> ?! Was wondering why discord bots for recent scores stopped working for unranked, thought  that was intentional

This has not changed and was implemented incorrectly from the beginning.
